cdp4dal/src/main/java/cdp4dal/Utils.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Utils.java
+ *
+ * Copyright (c) 2015-2019 RHEA System S.A.
+ *
+ * Author: Alex Vorobiev, Yevhen Ikonnykov, Sam Gerené
+ *
+ * This file is part of CDP4-SDKJ Community Edition
+ *
+ * The CDP4-SDKJ Community Edition is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * The CDP4-SDKJ Community Edition is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package cdp4dal;
+
+import cdp4common.dto.OwnedThing;
+import cdp4common.dto.ParameterBase;
+import cdp4dal.operations.OperationKind;
+import com.google.common.base.Strings;
+import java.net.URI;
+
+/**
+ * The static helper class that provides utilities to assist the Data Access Layer.
+ */
+public class Utils {
```

Comment:
@IevhenIkonnykov looking at the source code of the C# SDK I have noticed that this class has become a bit of a catchall for helper methods which is not clean at all. I created a isse https://github.com/RHEAGROUP/CDP4-SDK-Community-Edition/issues/47 that captures an improvement for this. I propose you implement this improvement already in the Java  SDK.